### PR TITLE
Add the possibility to remove the server splash

### PR DESCRIPTION
**Pull request by Vampire**

### CODE REVIEW

File: src/main/java/de/btobastian/javacord/entities/ServerUpdater.java [hunk 1]
```diff
@@ -551,7 +576,10 @@ public ServerUpdater removeAllRolesFromUser(User user, Collection<Role> roles) {
         if (icon != null) {
             patchServer = true;
         }
-        if (splash != null) {
+        if (updateSplash) {
+            if (splash != null) {
+                body.putNull("splash");
```
Comment (BtoBastian) [inline src/main/java/de/btobastian/javacord/entities/ServerUpdater.java]: I'm pretty the if statement here is wrong

Comment (Vampire) [reply src/main/java/de/btobastian/javacord/entities/ServerUpdater.java]: argh, copy & paste
1 sec

Comment (Vampire) [reply src/main/java/de/btobastian/javacord/entities/ServerUpdater.java]: fixed
that's the problem if you cannot test :-D

Comment (BtoBastian) [reply src/main/java/de/btobastian/javacord/entities/ServerUpdater.java]: That's why reviewing exists :-)

**Pull request merged**